mw-webapp/src/component/progressBar/ProgressBar.module.scss

```diff
@@ -19,7 +20,11 @@ $progressBarBorderRadius: 20px;
 }
 
 .progressIndicatorPercent {
+  position: absolute;
+  right: 0;
+  left: 0;
   margin-right: $marginSmall;
   margin-left: $marginSmall;
   color: var(--primaryTextColor);
+  text-align: center;
```

Comment:
text should be placed in the center of the progress bar. Does it works now this way?